### PR TITLE
completed assignment

### PR DESCRIPTION
**Pull request by TamaHills**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -14,29 +14,124 @@ const siteContent = {
...
+
+let navs = document.getElementsByTagName('nav');
+let nav = navs[0];
+let navItems = [...nav.children];
+navItems.forEach((item, i) => {
+  let content = document.createTextNode(siteContent['nav'][`nav-item-${i + 1}`]);
+  item.appendChild(content);
+  item.style.color = "green";
+});
+
```
Comment (rleslie1015) [inline js/index.js]: **Awesome job** looping through the nodelist you have selected and assigning values from the siteContent object!!!

File: js/index.js [hunk 2]
```diff
@@ -14,29 +14,124 @@ const siteContent = {
...
+let newLink = document.createElement('A');
+let newText = document.createTextNode('Hello');
+newLink.append(newText);
+newLink.href = "#"
+newLink.style.color = "green"
+
```
Comment (rleslie1015) [inline js/index.js]: Great job using append to add a nav item!

File: js/index.js [hunk 3]
```diff
@@ -14,29 +14,124 @@ const siteContent = {
...
+let ctaText = document.getElementsByClassName('cta-text')[0];
+let ctaHeading = ctaText.getElementsByTagName('H1')[0];
+let ctaButton = ctaText.getElementsByTagName('BUTTON')[0];
+cta['h1'].split(' ').forEach(i => {
+  let text = document.createTextNode(i);
+  let lb = document.createElement('BR');
+  ctaHeading.appendChild(text);
+  ctaHeading.appendChild(lb);
+})
+
```
Comment (rleslie1015) [inline js/index.js]: Awesome job adding line breaks with javascript!

File: js/index.js [hunk 4]
```diff
@@ -14,29 +14,124 @@ const siteContent = {
...
+// STRETCH BUTTON
+
+let rotate = document.createElement('BUTTON');
+rotate.appendChild(document.createTextNode("rotate"))
+rotate.addEventListener('click', () => {
+  mainContent.unshift(mainContent.pop());
+  mainContent.unshift(mainContent.pop());
+  applyContent(mainContent);
+});
+
```
Comment (rleslie1015) [inline js/index.js]: awesome job on the stretch!

**Comment by rleslie1015** [comment]
My only advice is when adding functions make sure they are still readable for other dev's and yourself in the future. I love that you're playing with javascript functions but sometimes simplicity is preferred. In this case, I think your code was a bit overkill. Still impressive that it all works though. :)

**Comment by TamaHills** [comment]
Appreciate the feedback! You're very right not only was it overly complicated I wasted too much time on it. That's what i get for trying to be "clever".

**Pull request merged**